docs/quickstart.rst

```diff
@@ -29,11 +30,47 @@ Create *quickstart.py* file and copy and paste the following code.
     gauth = GoogleAuth()
     gauth.LocalWebserverAuth() # Creates local webserver and auto handles authentication.
 
-Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
+Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. 
+Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
 
 .. _`APIs Console`: https://console.developers.google.com/iam-admin/projects
 .. _`OAuth made easy`: ./oauth.html
 
+Authentication via service accounts
+-----------------------------------
+
+Alternatively to OAuth2.0 client authentication *PyDrive2* can use Google Service Accounts for authentication.
```

Comment:
Make Google Service Accounts a link to general docs about service accounts?

Reply:
https://cloud.google.com/iam/docs/service-accounts